SPEC.md

```diff
@@ -118,32 +122,60 @@ An optional *path whitelist* can be provided, in which case all non-specified fi
 
 Image Format TODO
 
-* Define the garbage collection lifecycle of the container filesystem including:
+* Define the garbage collection lifecycle of the pod filesystem including:
     * Format of app exit code and signal
     * The refcounting plan for resources consumed by the ACE such as volumes
-* Define the lifecycle of the container as all exit or first to exit
-* Define security requirements for a container. In particular is any isolation of users required between containers? What user does each application run under and can this be root (i.e. "real" root in the host).
+* Define the lifecycle of the pod as all exit or first to exit
+* Define security requirements for a pod. In particular is any isolation of users required between pods? What user does each application run under and can this be root (i.e. "real" root in the host).
 * Define how apps are supposed to communicate; can they/do they 'see' each other (a section in the apps perspective would help)?
 
 
 ## App Container Executor
 
-App Containers are a combination of a number of technologies which are not aware of each other.
-This specification attempts to define a reasonable subset of steps to accomplish a few goals:
-
-* Creating a filesystem hierarchy in which the app will execute
-* Running the app process inside of a combination of resource and namespace isolations
-* Executing the application inside of this environment
-
+The **App Container Executor** defines the process by which applications contained in ACIs are executed.
 There are two "perspectives" in this process.
-The "*executor*" perspective consists of the steps that the container executor must take to set up the containers. The "*app*" perspective is how the app processes inside the container see the environment.
+The "*executor*" perspective consists of the steps that the App Container Executor (ACE) must take to set up the environment for the pod and applications.
+The "*app*" perspective is how the app processes inside the pod see the environment.
 
 ### Executor Perspective
 
+#### Pod Manifest
+
+The deployable, executable unit in the App Container specification is the **pod**.
```

Comment:
> It might be more clear to introduce the pod concept separately from the manifest and before introducing the details of the executor. A small top-level section right before ## App Container Executor?

I am going back and forth on this, struggling to make it fit with the greater structure (e.g. the top level three key sections of the spec)